.travis.yml

```diff
@@ -1,25 +1,37 @@
-language: c
+language: cpp
```

Comment:
Are there any implication of changing compiler from gcc to g++ ? Does this prevent to compile on some older platforms ? 
@malaterre any advice on this ?